docs/process.Markdown

```diff
@@ -0,0 +1,38 @@
+Release Processes
+=================
+
+Minor version updates (1.X.Y to 1.X.Y+1)
+----------------------------------------
+
+When:
+
+ * Such an update ensures we clear the cache, so it should be done when a change is committed that requires that (for example, a change to libc or libc++).
+ * The emsdk compiled versions are based on the version number, so periodically we can do this when we want a new precompiled emsdk version to be available.
+
+Requirements:
+
+ * We should check that there is no obvious breakage on the github CI at least before.
+
+How:
+
+ * The emscripten, emscripten-fastcomp, and emscripten-fastcomp-clang repos should each be updated: the emscripten-version.txt file in each, and a git tag (with the simple version number).
+ * A tag should also be done in the binaryen repo.
+
+Major version update (1.X.Y to 1.X+1.0)
```

Comment:
Calling a second level number update a "major" version update is sure to confuse some people...

Reply:
Fair point. What's a better term? "intermediate" maybe?